x/batching/types/params.go

```diff
@@ -0,0 +1,17 @@
+package types
+
+const (
+	DefaultValSetTrimPercent uint32 = 95 // 95%
+)
+
+// DefaultParams returns default batching module parameters.
+func DefaultParams() Params {
+	return Params{
+		ValidatorSetTrimPercent: DefaultValSetTrimPercent,
+	}
+}
+
+// Validate validates the batching module parameters.
+func (p *Params) Validate() error {
+	return nil
```

Comment:
Do we want to check that the trimPercent does not exceed 100%?

Reply:
good point - fixed